tests/fixtures/test_config.yaml

```diff
@@ -0,0 +1,9 @@
+inbox_bucket: test-inbox
+s3_endpoint_url: http://localhost:4566
+s3_secret_access_key: test
+s3_session_token: test
```

Comment:
```suggestion
```
Do we need the session token? 
It's currently unused when setting up s3 as we have both secret + access key